templates/blog/article.html

```diff
@@ -7,7 +7,9 @@
 {# Change canonical URL for "snapcraft.io" articles (2996 is the ID of the "snapcraft.io" tag) #}
 
 {% block body_class %}blog-article{% endblock %}
-{% block canonical_url %}{% if 2996 in article.tags %}https://snapcraft.io/blog/{{ article.slug }}{% elif 4059 in article.tags %}https://juju.is/blog/{{ article.slug }}{% else %}{{ super() }}{% endif %}{% endblock canonical_url %}
+{% block canonical_url %}
+  {% if 2996 in article.tags %}https://snapcraft.io/blog/{{ article.slug }}{%- elif 4059 in article.tags -%}https://juju.is/blog/{{ article.slug }}{%- elif 1304 in article.tags or 3681 in article.tags -%}https://maas.io/blog/{{ article.slug }}{% else %}{{ super() }}{% endif %}
+{% endblock canonical_url %}
```

Comment:
It looks like breaking this block to multiple lines is causing line breaks to appear in the `href` value:
![Screenshot from 2022-03-24 17-08-09](https://user-images.githubusercontent.com/2376968/159971930-ed70c9b0-d517-4dee-8055-3e5ca0d1f029.png)

Reply:
Fixed